public/css/app.css

```diff
@@ -1,101 +1,115 @@
-button {
+/*=====================================================*/
+/*                     General CSS                     */
```

Comment:
as long as we're sectioning things off, we might as well split this into separate files and include them as needed. if that's too much work though, don't worry about it - a little late in the game to be doing big rewrites lol